app/styles/_core.less

```diff
@@ -164,6 +164,12 @@ body.ios {
   }
   .btn {
     margin: 10px 0 20px;
+    &.clear-filter {
+      font-size: inherit;
+      font-weight: inherit;
+      vertical-align: initial;
+      padding: 0px;
```

Comment:
No need for `px` for zero values.

```
padding: 0;
```

Reply:
I'd rather make this style generic, something like `inline-btn-link` and target `btn-link` specifically.

```css
.btn-link.inline-btn-link {
   ...
}
```

Then we can reuse it in other places.